rules/macos/privilege_escalation_explicit_creds_via_apple_scripting.toml

```
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2020/12/07"
+maturity = "production"
+updated_date = "2020/12/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies execution of the security_authtrampoline process via the Apple Scripting Interpreter (osascript). This occurs
+when programs use AuthorizationExecute-WithPrivileges from the Security.framework to run another program with root
+privileges. It should not be run by itself, as this is a sign of execution with explicit logon credentials.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Execution with Explicit Credentials via Apple Scripting"
```

```suggestion
name = "Execution using Explicit Credentials via Scripting"
```
I dont think we need to specify `Apple` in each rule name since it is already tagged for macOS, but lgtm

---

agree but scripting includes also python and shell scripts, prefer to make it specific to AppleScript and JXA